src/auditor/compiler_abi.jl

```diff
@@ -158,8 +158,8 @@ function detect_cxxstring_abi(oh::ObjectHandle, platform::Platform; io::IO = std
         if isa(e, InterruptException)
             rethrow(e)
         end
-        warn(io, "$(path(oh)) could not be scanned for cxx11 ABI!")
-        Base.display_error(io, e, [])
+        @warn("$(path(oh)) could not be scanned for cxx11 ABI!")
```

Comment:
You can use `exception` to attach the exception:
```suggestion
        @warn("$(path(oh)) could not be scanned for cxx11 ABI!") exception=e
```
optionally with the backtrace like this:
```suggestion
        @warn("$(path(oh)) could not be scanned for cxx11 ABI!") exception=(e, catch_backtrace())
```

Reply:
WOW I LOVE IT.